apple-app-site-association.json

```diff
@@ -25,21 +27,22 @@
           "NOT /2016-year-in-art",
           "NOT /venice-biennale*",
           "NOT /gender-equality*",
-          "NOT /collections",
-          "NOT /collection/*",
           "NOT /click/*/aHR0cHM6Ly9pdHVuZXMuYXBwbGUuY29tL3VzL3BvZGNhc3QvYXJ0c3kv*",
           "NOT /click/*/aHR0cHM6Ly9hcnRzeS5uZXQv*",
           "NOT /click/*/aHR0cHM6Ly9hcnRzeS50eXBlZm9ybS5j*",
           "NOT /click/*/aHR0cHM6Ly9hcHAuYWRqdXN0LmNv*",
           "NOT /users/auth/facebook/callback*",
+          "NOT **",
           "NOT *L2FydGljbGUv*",
+          "NOT **",
           "NOT *L3ZpZGVv*",
           "NOT *L25l*",
           "NOT *L25ld3Mv*",
+          "NOT **",
```

Comment:
I don't know why these keep showing up in the file, but we should remove `NOT **` entries.

Reply:
This is a bug in the `yarn build` script for this repo. Just got these extraneous `NOT **` after running it.

Reply:
Yeah, I need to dig into that 😅